nat-lab/tests/utils/connection/docker_connection.py

```diff
@@ -39,9 +39,7 @@ async def mapped_ports(self) -> tuple[str, str]:
         details = await self._container.show()
         ports = details["NetworkSettings"]["Ports"]
```

Comment:
Don't entirely understand how is pyro port determined now

Reply:
pyro selects the port given by the OS, and then it is printed in [stdout](https://github.com/NordSecurity/libtelio/blob/main/nat-lab/tests/uniffi/libtelio_remote.py#L193), and later collected by the other pytest code

Reply:
I added a comment about this here: https://github.com/NordSecurity/libtelio/pull/696/files#diff-f1122f50402db2f1613c75eb1db5aed7bc4c9ed907b99f1bf4e5010345bfe6cdR629